code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -235,7 +235,7 @@ nobiliumsuppression = INFINITY
 			else
 				ASSERT_GAS(/datum/gas/carbon_dioxide,air)
 				ASSERT_GAS(/datum/gas/water_vapor,air)
-				cached_gases[/datum/gas/carbon_dioxide][MOLES] += plasma_burn_rate
+				cached_gases[/datum/gas/carbon_dioxide][MOLES] += plasma_burn_rate * 0.75
```

Comment:
in my current PR i changed this to 0.05, but this should be better

Reply:
Aight thanks for mentioning it :)